app/lib/utils/translations/translations_transaction.dart

```diff
@@ -0,0 +1,132 @@
+abstract class TranslationsTransaction {
+  String get endorseNewcomerNotificationTitle;
+  String get endorseNewcomerNotificationBody;
+  String get registerParticipantNotificationTitle;
+  String get registerParticipantNotificationBody;
+  String get unregisterParticipantNotificationTitle;
+  String get unregisterParticipantNotificationBody;
+  String get attestNotificationTitle;
+  String get attestNotificationBody;
+  String get claimRewardsNotificationTitle;
+  String get claimRewardsNotificationBody;
+  String get balanceTransferNotificationTitle;
+  String get balanceTransferNotificationBody;
+}
+
+class TranslationsEnTransaction implements TranslationsTransaction {
+  @override
+  String get endorseNewcomerNotificationBody => 'Thanks for endorsing the newbie!';
+  @override
+  String get endorseNewcomerNotificationTitle => 'Newbie endorsed';
+  @override
+  String get registerParticipantNotificationBody => 'You will receive a reminder one day before.';
+  @override
+  String get registerParticipantNotificationTitle => 'Registered for the next cycle!';
+  @override
+  String get attestNotificationBody =>
+      'If all participants have sent the attestations, you can try to claim the income.';
+  @override
+  String get attestNotificationTitle => 'Attested attendees';
+  @override
+  String get claimRewardsNotificationBody => 'You have already received your community income!';
+  @override
+  String get claimRewardsNotificationTitle => 'Claimed the community income';
+  @override
+  String get balanceTransferNotificationBody => 'The recipient has received the tokens.';
+  @override
+  String get balanceTransferNotificationTitle => 'Transaction completed';
+  @override
+  String get unregisterParticipantNotificationBody =>
+      'Your registration for the next cycle has been cancelled. If you change your mind, you can register again.';
+  @override
+  String get unregisterParticipantNotificationTitle => 'Registration Cancelled';
```

Comment:
`cancelled should be uncapitalized like in the other translations.

```suggestion
  String get unregisterParticipantNotificationTitle => 'Registration cancelled';
```